src/features/movies/moviesSaga.js

```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
+import { takeLatest, put } from "redux-saga/effects";
+import { fetchError, fetchMoviesSuccess, fetchMovieSuccess } from "./moviesSlice";
+import { apiUrl, apiKey } from "../../apiData";
+
+function* fetchMoviesHandler() {
+    try {
+        const data = yield axios.get(
+            `${apiUrl}movie/popular?api_key=${apiKey}&language=en-US&page=1`
```

Comment:
for this, i would rather make a fn asyn/await in other js file for api and then use it in saga

Reply:
ok, we can do this with async/await and add other file